### PR TITLE
Issue 7 A user should be able to read its mails

### PR DESCRIPTION
**Pull request by chibenwa**
As a bonus, I offer flags updates.

Based https://github.com/linagora/james-gatling

Solves https://github.com/linagora/james-gatling/issues/7

### CODE REVIEW

File: src/test/scala-2.11/org/apache/james/gatling/jmap/scenari/JmapGetMessageListScenario.scala [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import scala.concurrent.duration._
+
+class JmapGetMessageListScenario extends Simulation {
+
+
+  val userCount = 100
+
+  val feeder = UserFeeder.createUserFeederWithInboxAndOutbox(userCount)
+
+  val scn = scenario("JmapGetMessagesLists")
```
Comment (mbaechler) [inline src/test/scala-2.11/org/apache/james/gatling/jmap/scenari/JmapGetMessageListScenario.scala:17]: scenario are composable, please do not copy/paste from previous scenario

File: src/main/scala-2.11/org/apache/james/gatling/jmap/JmapMessages.scala [hunk 2]
```diff
@@ -60,4 +60,35 @@ object JmapMessages {
       .check(jsonPath("$.error").notExists)
       .check(jsonPath("$[0][1].messageIds[*]").findAll.saveAs("messageIds"))
 
+  def getRandomMessage() =
+    JmapAuthentication.authenticatedQuery("getMessages", "/jmap")
+      .body(StringBody(
+        """[[
+          "getMessages",
+          {
+            "ids": ["${messageIds.random()}"],
```
Comment (mbaechler) [inline src/main/scala-2.11/org/apache/james/gatling/jmap/JmapMessages.scala:71]: please, no code in strings

File: src/main/scala-2.11/org/apache/james/gatling/jmap/JmapMessages.scala [hunk 3]
```diff
@@ -91,4 +91,26 @@ object JmapMessages {
       .check(status.is(200))
       .check(jsonPath("$.error").notExists)
 
+  def markAsRead() = performUpdate("markAsRead", "isUnread", "false")
```
Comment (mbaechler) [inline src/main/scala-2.11/org/apache/james/gatling/jmap/JmapMessages.scala]: strings everywhere ! you can do much better, use types, compose, etc

File: src/main/scala-2.11/org/apache/james/gatling/jmap/JmapMessages.scala (hunk 2 above)
Comment (chibenwa) [inline src/main/scala-2.11/org/apache/james/gatling/jmap/JmapMessages.scala:71]: It is not scala code. It is not a s"" string.

Here you see Gatling DSL on session access. 

By the way, there is no other way to do it.

**Comment by rouazana** [comment]
rebased here: #20

**Pull request closed without merging**